### ISSUE TITLE
Wallet: Reenable -fallbackfee by default for regtest and signet (and maybe testnet too)?

### ISSUE THREAD

**Issue by michaelfolkson** [bitcoin/bitcoin#20087]
This [PR](https://github.com/bitcoin/bitcoin/pull/16524) merged in October 2019 disabled `-fallbackfee` by default across all chains (mainnet, testnet, regtest). 

The only motivation for doing this for regtest (and maybe testnet too) that I can see is:

> Now it is 0 by default for all chains, thus there's no need to call Params(). ([comment](https://github.com/bitcoin/bitcoin/pull/16524#issue-303550511))

However, this does mean that if you don't set a fee or enable `-fallbackfee` you get this error.

```
error code: -4
error message:
Fee estimation failed. Fallbackfee is disabled. Wait a few blocks or enable -fallbackfee.
```
Any thoughts on reenabling `-fallbackfee` by default for regtest, signet (and maybe testnet too)? I think it makes sense to continue to have it disabled by default for real money on mainnet.

If I'm missing context please direct me to it and I'll close this.

Bitcoin StackExchange post: https://bitcoin.stackexchange.com/questions/97174/when-using-bitcoin-cli-i-get-an-error-regarding-fallback-fees-when-trying-to-sen

**Comment by MarcoFalke** [comment]
No strong opinion, but I have a slight preference to keep it the way it is. If documentation is missing, then it should be improved. The error message could probably be updated on how to set a feerate over RPC, not just the startup setting, which would require a restart of the server.

**Comment by michaelfolkson** [comment]
So an extended error message, something like:

`Fee estimation failed and -fallbackfee is disabled. Enable -fallbackfee (requires restart), wait a few blocks or manually set a fee as an additional argument to your bitcoin-cli command e.g. 1 sat/B`

You don't want **too** long error messages right?

**Comment by MarcoFalke** [comment]
> You don't want too long error messages right?

Right. Maybe it is time to explain Bitcoin Core fee estimation/fee settings to users in one document and refer to that in the error message?

**Comment by michaelfolkson** [comment]
Yeah I think the above is the best approach longer term. I'll close this for now. Thanks

**Comment by MarcoFalke** [comment]
Heh, this is still something that should be addressed. ;)

**Comment by michaelfolkson** [comment]
Ok I'm happy for this to be a good first issue or to do it myself.

For now here are some resources that will be useful.

How Bitcoin Core estimates transaction fee ([StackExchange](https://bitcoin.stackexchange.com/questions/96489/how-bitcoin-core-estimates-transaction-fee))

Bitcoin Code code documentation on [BlockPolicyEstimator](https://github.com/bitcoin/bitcoin/blob/195822f1e05e2f36002c906667d4c639663f23b5/src/policy/fees.h#L83)

The fallback fee error ([StackExchange](https://bitcoin.stackexchange.com/questions/97174/when-using-bitcoin-cli-i-get-an-error-regarding-fallback-fees-when-trying-to-sen)

Presumably you mean something like this PSBT [doc](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md) but for fee estimation/fee settings. And stored in the doc directory.
